reference/commands/misc/inspect.rst

```diff
@@ -0,0 +1,52 @@
+
+.. _conan_inspect:
+
+conan inspect
+=============
+
+.. note::
+
+    This is an EXPERIMENTAL new command in Conan 1.8, and might be subject to changes in future releases.
+
+.. code-block:: bash
+
+    $ .. code-block:: bash
+
+    $ conan inspect [-h] [-a [ATTRIBUTE]] [-r REMOTE] [-j JSON]
+                     path_or_reference
+
+Displays conanfile attributes, like name, version, options Works both locally,
+in local cache and remote
+
+.. code-block:: text
+
+    positional arguments:
+      path_or_reference     Path to a folder containing a recipe (conanfile.py) or
+                            to a recipe file. e.g., ./my_project/conanfile.py. It
+                            could also be a reference
+
+    optional arguments:
+      -h, --help            show this help message and exit
+      -a [ATTRIBUTE], --attribute [ATTRIBUTE]
+                            The attribute to be displayed, e.g "name"
+      -r REMOTE, --remote REMOTE
+                            look in the specified remote server
+      -j JSON, --json JSON  json output file
+
+
+Example:
+
+.. code-block:: bash
+
+    $ conan inspect zlib/1.2.11@conan/stable -a=name -a=version -a=options -a default_options -r=conan-center
+    name: zlib
+    version: 1.2.11
+    options
+        shared: [True, False]
+    default_options: shared=False
+
+.. code-block:: bash
```

Comment:
I will include an example with the output of an recipe inspection stored in a remote to show it actually downloads the recipes

Reply:
What happens when no attribute is specified? could it be a good example?

Reply:
thanks!

Reply:
Done:
- Removed the extra code-block
- Fixed command, so it outputs some default fields in case no ``-a`` is specified.

@danimtb An example of the output of a remote is already shown